eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -61,8 +61,8 @@ function Should-Process-Package($pkgPath, $packageName)
     # Get package info from json file created before updating version to daily dev
     $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
     $packagePath = $pkgInfo.DirectoryPath
-    $modifiedFiles = git diff --name-only --relative $TargetBranch HEAD
-    $modifiedFiles = $modifiedFiles.Where({$_.startswith($packagePath)})
+    $gitDiffChanges = Join-Path $PSScriptRoot "git-diff-changes.ps1"
```

Comment:
$gitDiffChanges doesn't seem like the right name here as it is really the gitdiffChangesScriptPath. Given a lot of usages are other PS scripts we should actually consider making this a helper function instead. It will make it easier to consume.